content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,211 @@
+# Application Registration Prerequisites / Developer Guide
+
+## Audience
+
+This technical specification is targeted at software developers building applications on top of Cerner's Ignite application platform as part of the [Cerner CODE ecosystem](https://code.cerner.com/).
```

Comment:
"Cerner CODE ecosystem", Cerner is the "C" in CODE so it's a bit redundant...

Also, usually it's referred to as the code program, not ecosystem which is used later on. Also it's usually "code" not "CODE"

Reply:
FWIW - I agree that Cerner is redundant and can be removed. Though, I will say that I think it's quite silly that Cerner goes through the trouble of making a cute acronym and then intentionally downplays the fact that is is an acronym by electing to display it in lower case.  What's the point?   Too bad this isn't an actual Wiki, so we could let the Doc Developers could handle this with their "protect the brand" magic-templates...

Reply:
I removed "Cerner" and changed "CODE" to "code" in the whole doc.